### PR TITLE
Update footer links

### PR DESCRIPTION
**Pull request by tbille**
## Done
- Update footer links to point somewhere :laughing: 

## How to QA
- `dotrun`
- http://localhost:8041
- Make sure footer is still looking good and that the links link to somewhere. (some links are still missing)

### CODE REVIEW

File: templates/footer.html [hunk 1]
```diff
@@ -7,23 +7,23 @@ <h2 class="p-heading--5">Charmed Operator Framework</h2>
 
       <ul class="p-list">
         <li class="p-list__item">
-          <a class="p-link--soft" href="https://juju.is"><small>What is Juju?</small></a>
+          <a class="p-link--soft" href="/"><small>What is Juju?</small></a>
```
Comment (anthonydillon) [inline templates/footer.html:10]: Probably want an id to anchor to the what is Juju section?

File: templates/footer.html [hunk 2]
```diff
@@ -167,7 +163,7 @@ <h2>Join the community</h2>
           <a class="p-link--soft js-revoke-cookie-manager" href=""><small>Manage your tracker settings</small></a>
         </li>
         <li class="p-list__item">
-          <a class="p-link--soft" href="https://github.com/canonical-web-and-design/juju.is/issues/new"><small>Report a bug on this site</small></a>
+          <a class="p-link--soft" href="https://github.com/canonical-web-and-design/charmhub.io/issues/new"><small>Report a bug on this site</small></a>
```
Comment (anthonydillon) [inline templates/footer.html:166]: Juju.is right?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Done comments in the code

File: templates/footer.html (hunk 2 above)
Comment (tbille) [reply templates/footer.html]: bad copy/paste. Good catch.

File: templates/footer.html (hunk 1 above)
Comment (tbille) [reply templates/footer.html]: Done

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**